src/main/java/com/alkemy/ong/controller/CategoryController.java

```diff
@@ -35,6 +35,7 @@
         }
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    //Create category
```

Comment:
It would be useful to add the permission as a comment
For example
// Create category method only for ADMIN
what do you think?

Reply:
Good idea. Added.